src/w3c/defaults.js

```diff
@@ -10,14 +10,16 @@ import { rule as localRefsExist } from "../core/linter-rules/local-refs-exist";
 import { rule as noHeadinglessSectionsRule } from "../core/linter-rules/no-headingless-sections";
 import { rule as noHttpPropsRule } from "../core/linter-rules/no-http-props";
 import { rule as privsecSectionRule } from "./linter-rules/privsec-section";
+import { rule as checkCharset } from './linter-rules/check-charset';
 
 linter.register(
   noHttpPropsRule,
   privsecSectionRule,
   noHeadinglessSectionsRule,
   checkPunctuation,
   localRefsExist,
-  checkInternalSlots
+  checkInternalSlots,
+  checkCharset
```

Comment:
Oops. would you move this to src/core/defaults.js? Before #2042 this was a right change but we decided to move to core/defaults.

Reply:
Ah, I don't have the `src/core/defaults.js` file ... it just got merged , I'll have to pull first